FoodSuggestions/app/src/main/java/com/example/foodsuggestions/adapters/RecipeAdapter.java

```diff
@@ -44,39 +49,55 @@ public ViewHolder(View itemView) {
             nameRecipe = (TextView) itemView.findViewById(R.id.idTitleRe);
             imageRecipe = (ImageView) itemView.findViewById(R.id.idImageRe);
 
-            itemView.setOnClickListener(this);
```

Comment:
Please set the click listener in onCreateViewHolder. See https://developer.android.com/reference/androidx/recyclerview/widget/RecyclerView.ViewHolder#getBindingAdapterPosition()